modules/ocf_irc/manifests/ircd.pp

```diff
@@ -0,0 +1,32 @@
+class ocf_irc::ircd {
+  package { 'inspircd':; }
+
+  service { 'inspircd':
+    ensure  => 'running',
+    restart => 'service inspircd reload',
+    require => Package['inspircd'],
+  }
```

Comment:
I feel that `ensure => running` is a smell in Puppet code. Is there a reason Puppet needs to start this service? I'd think systemd should be able to handle it.

Also, why changing `restart`?